RInAppMessaging/Classes/Router.swift

```diff
@@ -6,8 +6,11 @@ internal protocol RouterType: AnyObject {
     /// Contains logic to display the correct view type and create
     /// a view controller to present a single campaign.
     /// - Parameter campaign: The campaign object to display.
+    /// - Parameter confirmation: A handler called just before displaying.
     /// - Parameter completion: Completion handler called once displaying has finished.
-    func displayCampaign(_ campaign: Campaign, completion: @escaping () -> Void)
+    func displayCampaign(_ campaign: Campaign,
+                         confirmation: @escaping () -> Bool,
```

Comment:
I think that you can add `@autoclosure`here so instead of having:
`confirmation: { true }` or `confirmation: { false }`
We would have:
`confirmation: true` or `confirmation: false`

But that would mean to call a function instead of a closure for a complex code.

Reply:
I had a problem with complex code in CampaignDispatcher but I solved it just by adding `()` at the and so yeah I think I like this better now~